lib/gmail-adapter.js

```diff
@@ -80,6 +89,10 @@ var Read = Juttle.proc.base.extend({
         // Build a search expression given the options.
         var search = self.raw || "";
 
+        if (this.filter_search_expr !== '') {
```

Comment:
That if avoids adding an additional space, but I'll initialize it to undefined instead.